lib/parser.js

```diff
@@ -91,14 +91,32 @@ module.exports = function (content) {
   cb(null, 'module.exports = ' + JSON.stringify(output))
 }
 
-function commentScript (content) {
+function commentScript (content, lang) {
   return content
     .split(/\n\r|\n|\r/g)
     .map(function (line) {
-      if (line.trim() === '') {
-        return line
-      } else {
-        return '// ' + line
+      switch (lang) {
```

Comment:
I think we can check for `line.trim()` outside the switch, since it's the same case for all languages.